.github/workflows/reusable-docker-publish.yml

```
@@ -0,0 +1,113 @@
+        default: ''
+      dry_run:
+        description: 'If true, skip release creation (validation mode)'
+        required: false
+        type: boolean
+        default: false
+      build_logic_ref:
+        description: 'Ref for liquibase/build-logic checkout'
+        required: false
+        type: string
+        default: 'main'
+    secrets: inherit
```

actionlint (1.7.12)

[error] "secrets" section is scalar node but mapping node is expected

(syntax-check)